spacy/lang/ja/__init__.py

```diff
@@ -43,42 +48,22 @@ def resolve_pos(token, next_token):
     resolve ambiguous mappings.
     """
 
-    # this is only used for consecutive ascii spaces
-    if token.surface == " ":
-        return "空白"
-
     # Some tokens have their UD tag decided based on the POS of the following
     # token.
-    next_pos = None
-    if next_token:
-        next_pos = next_token.pos
 
-    if token.pos == "連体詞":
-        if re.match(r"[こそあど此其彼]の", token.surface):
-            return token.pos + "-DET"
-        elif re.match(r"[こそあど此其彼]", token.surface):
-            return token.pos + "-PRON"
-        else:
-            return token.pos + "-ADJ"
+    # orth based rules
+    if token.pos in TAG_ORTH_MAP:
+        orth_map = TAG_ORTH_MAP[token.pos[0]]
+        if token.surface in orth_map:
+            return orth_map[token.surface]
```

Comment:
Does this actually work for 連体詞? The map looks like this:

    "こ": PRON,

But こんな is one token so `token.surface in orth_map` is never true. There's also no mapping for DET for この etc.

Reply:
Sorry, I'm not sure about these examples. I only implemented to keep the spec of old (maybe since v1.x) codes.
Let us reduce irrelevant rules.
I'd like to check the existance of them in `ja_gsd-train.conllu`. 
Just a moment, please.

Reply:
Sorry, were you going to follow up on this? If not I can go ahead and accept the PR anyway, just wanted to check.

Reply:
I changed the default pos for `連体詞` from DET to ADJ and added some lexical mapping entries to `tag_orth_map.py`.
These changes are based on the statistics below.
```
$ egrep $'\t'連体詞 ja_gsd-ud-train.conllu | cut -f 2,4,5 | sort | uniq -c
   1 わが	ADJ	連体詞
   9 主な	ADJ	連体詞
   1 去る	ADJ	連体詞
  45 同じ	ADJ	連体詞
   2 我が	ADJ	連体詞
   8 あの	DET	連体詞
 371 この	DET	連体詞
 425 その	DET	連体詞
  18 どの	DET	連体詞
  32 ある	PRON	連体詞
   1 おもな	ADJ	連体詞
   2 とある	ADJ	連体詞
   3 ほんの	ADJ	連体詞
  37 大きな	ADJ	連体詞
  10 小さな	ADJ	連体詞
   4 更なる	ADJ	連体詞
  12 こんな	PRON	連体詞
  21 そんな	PRON	連体詞
   8 どんな	PRON	連体詞
   1 いかなる	ADJ	連体詞
   5 いろんな	ADJ	連体詞
   4 いわゆる	ADJ	連体詞
   1 おおきな	ADJ	連体詞
   1 おかしな	ADJ	連体詞
   3 さらなる	ADJ	連体詞
   1 ちいさな	ADJ	連体詞
   2 ひょんな	ADJ	連体詞
   1 大いなる	ADJ	連体詞
   3 あらゆる	PRON	連体詞
   1 ちっちゃな	ADJ	連体詞
```

Reply:
Sorry. I wrote above comment to difference thread.
I'm still tuning the rules in `tag_bigram_map.py`.
Just a moment, please.